docs/web3.main.rst

```diff
@@ -49,6 +49,85 @@ Attributes
        'Geth/v1.4.11-stable-fed692f6/darwin/go1.7'
 
 
+.. _batch_requests:
+
+Batch Requests
+~~~~~~~~~~~~~~
+
+.. py:method:: Web3.batch_requests()
+
+    The JSON-RPC API allows for batch requests, meaning you can send a single request
+    that contains an array of RPC commands. Generally, this may be useful when you want
```

Comment:
```suggestion
    that contains an array of requests. Generally, this may be useful when you want
```

Reply:
```suggestion
    that contains an array of JSON-RPC objects. Generally, this may be useful when you want
```
??

Reply:
spec [docs](https://www.jsonrpc.org/specification#batch) refer to them as "request objects" so rolling with that.